tests/lib/hardware/hardware_base.py

```diff
@@ -159,10 +159,12 @@ def ansible_run_playbook(self, playbook: str,
         else:
             extra_vars_param = ""
 
+        extra_args = settings.get('ANSIBLE_PLAYBOOK_EXTRA_ARGS', '')
```

Comment:
- I would prefer to not add another variable that overlaps with `ANSIBLE_EXTRA_ARGS`
- If we go that way, it needs to be documented.

Reply:
I don't see any serious issues with overlaping, it gives user an option.

Reply:
It's confusing and everything you can do with `ANSIBLE_PLAYBOOK_EXTRA_ARGS` can be done already with `ANSIBLE_EXTRA_ARGS`. So what's the benefit?

Reply:
There two advantages:
1) allows multiple usage of --extra-vars, so you can, for example, use more than one json file with properties without prio combining it in one.
2) allows to provide other options to ansible-playbook rather than --extra-vars, see example in the doc (btw I added it by your request).